src/plugins/controls/public/control_group/editor/control_editor.tsx

```diff
@@ -259,7 +259,7 @@ export const ControlEditor = ({
               <EuiFormRow
                 label={ControlGroupStrings.manageControl.displaySettings.getWidthInputTitle()}
               >
-                <>
+                <div>
```

Comment:
Switched this to a `div` so that it can receive the `id` passed down from the `EuiFormRow` component - not sure why, but the React fragment ignored it 🤷